static/js/timerender.js

```diff
@@ -286,10 +286,7 @@ export const absolute_time = (function () {
         return str;
     };
 
-    return function (timestamp, today) {
-        if (typeof today === "undefined") {
-            today = new Date();
-        }
+    return function (timestamp, today = new Date()) {
```

Comment:
It's great that JS doesn't have the Python bug of evaluating default parameters only once at load time.